frontend/src/scenes/session-recordings/player/PlayerFrame.scss

```diff
@@ -10,6 +10,17 @@
     overflow: hidden;
     background-color: var(--bg-3000-dark);
 
+    .PlayerFrame__content--masking-window .replayer-wrapper::before {
```

Comment:
dump a image on top of the replayer
because rrweb controls the size and contents of the div we have to do it like this